src/Command/AnalyseApplication.php

```diff
@@ -35,9 +35,10 @@ public function __construct(Analyser $analyser, string $memoryLimitFile, FileHel
 	 * @param string[] $paths
 	 * @param \Symfony\Component\Console\Style\StyleInterface $style
 	 * @param bool $defaultLevelUsed
+	 * @param array $customExtensions
 	 * @return int
 	 */
-	public function analyse(array $paths, StyleInterface $style, bool $defaultLevelUsed): int
+	public function analyse(array $paths, StyleInterface $style, bool $defaultLevelUsed, array $customExtensions): int
```

Comment:
I'd like if the fileExtensions would be injected through the constructor since it's a DIC parameter and it's possible.